python/cuda_cccl/tests/parallel/test_unique_by_key_api.py

```diff
@@ -3,12 +3,14 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 
+import cupy as cp
+import numpy as np
+
+import cuda.cccl.parallel.experimental as parallel
+
+
```

Comment:
Don't think this change is needed

Reply:
https://github.com/NVIDIA/cccl/pull/5578#discussion_r2300778753